src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name, age = null, sex, isMarried, partnerName } = person;
```

Comment:
It is better to write each value on a new line if there are more than 3 values. This will improve the readability of the code.
```suggestion
  const { 
    name, 
    age, 
    sex, 
    isMarried, 
    partnerName 
  } = person;
```